features/steps/room_steps.js

```diff
@@ -1,9 +1,15 @@
 const { Given, When, Then } = require("cucumber");
+const WorkspacePage = require("../page-objects/WorkspacePage");
+const Workspace = require("../parameter-types/workspaces");
+const Room = require("../parameter-types/rooms");
 const assert = require('assert').strict;
 
-Given("a Workspace with {accessLevel} {room}", function (accessLevel, room) {
-  // Write code here that turns the phrase above into concrete actions
-  return "pending";
+Given("a Workspace with {accessLevel} {room}", async function (accessLevel, room) {
+  const workspace = new Workspace("System Test");
+  this.workspace = new WorkspacePage(this.driver, workspace);
+  await this.workspace.enter();
+  const lockedRooms = await this.workspace.roomCardsWhere({ accessLevel });
```

Comment:
Not blocking: I almost wish we had a `fluent interface` for this. I.e. `await this.workspace.enter().roomCardsWhere({ accessLevel })` but I think that may be hard to pull off.